### PR TITLE
WT-5645 Add Evergreen test that cycles through known failure test/format configs

### PR DESCRIPTION
**Pull request by lukech**
Adding a new Evergreen task as part of the major `ubuntu1804` build variant. The task calls a new shell script to cycle through a list of test/format CONFIGs that failed previously in various WT tickets, aiming for catching issues in a more targeted measure. 

New failure CONFIGs can be added easily (by adding a CONFIG.WT-xxxx file under the test/format/failure_configs directory) over time to extend testing coverage.

### CODE REVIEW

File: test/format/failure_configs/CONFIG.WT-5637 [hunk 1]
```diff
@@ -0,0 +1,103 @@
+############################################
+#  RUN PARAMETERS: V2
```
Comment (lukech) [inline test/format/failure_configs/CONFIG.WT-5637:2]: Converted the initial V1 CONFIG to V2, and disabled `backup.incremental`, `ops.rebalance`, and `ops.salvage` to pass the format run for this CONFIG.

Comment (sueloverso) [reply test/format/failure_configs/CONFIG.WT-5637:2]: FYI, as long as `backup=0` is set, it doesn't matter what the incremental setting is.

**Review by sueloverso — APPROVED** [review]
LGTM. What is the next step to dig into other tickets and pull their CONFIGs?
Also, how did you convert the CONFIG? Manually or automatically? The easiest way to convert it would be to run `test/format` with the old CONFIG, and then pull `RUNDIR/CONFIG` that it generates that is in the new format.

**Comment by lukech** [comment]
@sueloverso Instead of spending effort figuring out and adjusting CONFIGs from older tickets (which is time-consuming as some settings do not make sense in the latest version of test/format, based on the tweaking experience for WT-5637 CONFIG),  I prefer making it an ongoing effort so that in future team members can evaluate and add new CONFIGs when finding fits, as the testing scaffold has been set up in this PR. I will communicate with the team once the PR is merged.

In terms of CONFIG.WT-5637 conversion, I took the V2 version printed out on console (the format run failed as rebalance and salvage were initially set) and made small adjustments by disabling rebalance and salvage. And you are right I could have pull `RUNDIR/CONFIG` as it's with the new format.

**Review by tammybailey — APPROVED** [review]
lgtm

**Pull request merged**